src/components/application_manager/src/resumption/resumption_data_processor_impl.cc

```diff
@@ -534,9 +534,9 @@ void ResumptionDataProcessorImpl::DeleteSubmenus(
       application->app_id(), resumption_status_, resumption_status_lock_);
 
   auto accessor = application->sub_menu_map();
-  const auto& sub_menu_map = accessor.GetData();
+  const auto sub_menu_map = accessor.GetData();
 
-  for (const auto& smenu : sub_menu_map) {
+  for (const auto smenu : sub_menu_map) {
```

Comment:
```suggestion
  for (const auto& smenu : sub_menu_map) {
```

Reply:
fixed in e6e5767ef10ee27415d93deeac04dd30739b2f6e